pkg/assembler/backends/inmem/testhelpers.go

```diff
@@ -0,0 +1,147 @@
+package inmem
```

Comment:
wonder if this can be renamed to a `_test.go` file which will only be compiled if running a test.

Reply:
o wait this already exists, is this a duplicate file?